### ISSUE TITLE
Chocolatey package

### ISSUE THREAD

**Issue by robmonte** [SamsidParty/TopNotify#32]
Have you considered having a chocolately package version for releases? It would be nice instead of relying on Windows Store or coming to Github for the binary.

**Comment by SamsidParty** [comment]
TopNotify needs to be an MSIX package to work, so even if I added it to Choco then it would still have to be installed from the store.

**Comment by robmonte** [comment]
Would being a chocolatey package that still installs from the Windows Store enable it to work through automation? In my original request when I mentioned the Store, I was thinking more about the manual nature of installing it through there. However, if it can be automated with chocolatey while still coming from the Windows Store I think people may find that useful.

**Comment by SamsidParty** [comment]
Oh I see, I thought you were worried about the privacy / Microsoft account troubles of using the MS store. I will look into publishing to Choco/winget when I return from my trip.

**Comment by aqtoo** [comment]
I'd be interested in scoop too, but I'm unsure of the possibility considering scoops usage of mostly portable apps

**Comment by SamsidParty** [comment]
I think winget would be the best option considering it's built into windows.

**Comment by SamsidParty** [comment]
I have chosen winget for its native integration into windows and its support for MSIX files out of the box. I am currently working on submitting TopNotify to the winget repo.

**Comment by terax6669** [comment]
> TopNotify needs to be an MSIX package to work, so even if I added it to Choco then it would still have to be installed from the store.

Can you explain why that is?

**Comment by SamsidParty** [comment]
Actually I was wrong, MSIX packages do not need to be installed from the store

**Comment by terax6669** [comment]
Ok then let me rephrase my question.

Why does it have to be an MSIX in the first place? Is Microsoft locking access to some system APIs behind the store?

**Comment by SamsidParty** [comment]
Yes exactly, Microsoft is only allowing the notification listener API on MSIX/UWP apps, hence the "Not Packaged" error some people were getting.

**Comment by zeldrisho** [comment]
```powershell
scoop bucket add zeldrisho https://github.com/zeldrisho/scoop-bucket
scoop install zeldrisho/topnotify
```

```powershell
winget install SamsidParty.TopNotify
```
